scripts/fires/.gitattributes

```diff
@@ -0,0 +1 @@
+*.csv filter=lfs diff=lfs merge=lfs -text
```

Comment:
please also add the cached location_fille.json

Reply:
Done